### PR TITLE
Changed 2 files

### PR DESCRIPTION
**Pull request by vazgen4073**

### CODE REVIEW

**Comment by vazgen4073** [comment]
try_catch  main.cpp 37 row | throw  bank.cpp 121 row
try_catch  main.cpp 45 row | throw  bank.cpp 223 row
try_catch  main.cpp 53 row | throw  bank.cpp 353 row
try_catch  main.cpp 61 row | throw  bank.cpp 405 row
try_catch  main.cpp 69 row | throw  bank.cpp 452 row
---------------------------------------------------
try_catch  bank.cpp 259 row|
try_catch  bank.cpp 301 row|

File: bank.cpp [hunk 1]
```diff
@@ -248,8 +290,23 @@ void Bank::transactions()
 
         int amount;
 
+        std::system("clear");
         std::cout << "Enter the amount\n";
-        std::cin >> amount;
+        std::string str_amount;
```
Comment (haykaslanian) [inline bank.cpp]: It would be better to use integer type instead of std::string

File: main.cpp [hunk 2]
```diff
@@ -25,37 +25,58 @@ int main()
     while (!exit) {
         print_Welcome();
         std::cin >> num;
+        std::system("clear");
+
         switch (num) {
```
Comment (haykaslanian) [inline main.cpp]: It would be better to put whole switch in try block and after switch catch exceptions

Comment (vazgen4073) [reply main.cpp]: ok!

File: bank.cpp (hunk 1 above)
Comment (vazgen4073) [reply bank.cpp]: ok!

**Pull request merged**